pkgs/servers/web-apps/lemmy/ui.nix

```diff
@@ -33,15 +33,16 @@ let
     sha256 = pinData.uiSha256;
   };
 in
-mkYarnPackage {
+mkYarnPackage rec {
```

Comment:
Usage of `rec` like this cause people to use attributes as if they're mere let bindings, resulting in unused environment variables in the builder. This makes maintenance harder and causes unnecessary rebuilds sometimes.

Reply:
> resulting in unused environment variables in the builder

This is not the case here, all variables are in used by the called function. Furthermore, this is a problem of `mkDerivation` and not of recursive attrsets.

Most of the `mkYarnPackage` calls are already recursive attrsets, I'm not sure if one more or less matters that much. But I can put the `src` in a let binding if you want.